releases/release-1.11/release-1.11.md

```diff
@@ -0,0 +1,124 @@
+# Kubernetes 1.11 Release Schedule
+
+*Handy Links (not populated yet):*
+
+* [Release Team](http://bit.ly/kube111-team)
+* [Zoom](http://bit.ly/kube111-zoom)
+* [Slack](https://kubernetes.slack.com/messages/sig-release/)
+* [Forum](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
+* [Feature Tracking Sheet]()
+* [Milestone Process](https://github.com/kubernetes/community/blob/master/contributors/devel/release/issues.md)
+* [Burndown / Meeting Minutes](http://bit.ly/kube111-burndown)
+* [Retrospective Document](http://bit.ly/kube111retro)
+
+**tl;dr** The 1.10 release cycle begins on **Tuesday, April 3rd, 2018**, and ends on release day, **Wednesday, June 27th**.  Feature freeze is **Tuesday, April 24th**.  Code freeze begins **Monday, May 28th** and ends **Wednesday, June 20th**. Docs must be completed and reviewed by **Friday, May 11th**.
```

Comment:
1.11? also a few other places

Reply:
feh, need to do another search-and-replace

Reply:
will do after we get some other comments